### ISSUE TITLE
(Future Compatibility Issue): 

### ISSUE THREAD

**Issue by schluppeck** [justingardner/mrTools#21]
The `fullcrosshair` mouse pointer is deprecated in R2014b. Trying to use it causes a warning and then sets the pointer to `crosshair`. atmo, this only occurs in 3 files in the mrTools code:

    mrSearchHelp('fullcrosshair')
    ---------------------------------------------------------------------------
    talairach.m:102:     set(gTalairach.fig(a(1)),'pointer','fullcrosshair');
    talairach.m:407: set(gTalairach.fig(1),'pointer','fullcrosshair');
    talairach.m:408: set(gTalairach.fig(2),'pointer','fullcrosshair');
    talairach.m:409: set(gTalairach.fig(3),'pointer','fullcrosshair');
    mrInterrogator.m:133:   set(MLR.interrogator{viewNum}.fignum,'pointer','fullcrosshair');
    mlrVol.m:131:   set(gcf,'Pointer','fullcrosshair');
    ---------------------------------------------------------------------------

Using `crosshair` instead fixes this (but at the cost of not having the nice cross hairs).

**Comment by justingardner** [comment]
This is what mathworks considers to be an "upgrade"?

**Comment by schluppeck** [comment]
... there are so many things that are **a total pain** with this ;). I've just added a few fixes to the numeric/object handles issue so that `mrQuit()` works. `h.Number` can be empty, but I think the `mlrGetFignum()` spoofing a figure number that you implemented will work.

*Next pain:* reopening a `mrLastView.mat` file. If a graphics object (e.g. a figure) is saved in a .mat file, then the figure gets re-opened when you `load` the mat file. So the presence of a figure object in `view.figure` causes the following warning (and a bit of a mess when you restart `mrLoadRet`. I've added a line in `mrSaveView()` to deal with this. Pull request on the way...

    /data/retinotopyTutorial/mrLastView.mat. Loading this file
    recreates and displays the figure and its contents. Refer to
    save for more information.
